src/Tizen.NUI/src/public/Common/BaseHandle.cs

```diff
@@ -115,6 +115,7 @@ internal BaseHandle(global::System.IntPtr cPtr)
         /// Event when a property is set.
         /// </summary>
         /// <since_tizen> 5 </since_tizen>
```

Comment:
We could optionally also add a way to quickly jump to the new event.
```suggestion
        /// <since_tizen> 5 </since_tizen>
        /// <seealso cref="BindableObject.PropertyChanged" />
```

Reply:
thank you for your review.
it's added.